kpatch-build/kpatch-build

```diff
@@ -846,6 +846,9 @@ for i in $FILES; do
 		fi
 
 		readelf -s --wide "$KOBJFILE_PATH" > "$SYMTAB"
+		if [[ "$ARCH" = "ppc64le" ]]; then
+			sed -i 's/\[<localentry>: 8\]//' "$SYMTAB"
```

Comment:
```suggestion
			sed -ri 's/\s+\[<localentry>: 8\]//' "$SYMTAB"
```

I think we might want to eat up preceding whitespaces as well.

Reply:
Yeah, the extra whitespace tab... I wasn't sure how to handle both RHEL-8 and @kamalesh-babulal 's originally reported format cases.  One uses `\t[<localentry>: 8\]` and the other (I think) `[<localentry>: 8\]\t`.  Can you think of a regex that handles both cases without concatenating columns?

Reply:
I think only handling preceding tab case should be enough since we will never have `[<localentry>: 8]` at the beginning of the line.